src/components/interface/Table/Table.js

```diff
@@ -136,7 +136,7 @@ const Table = ({
             .map((row) => (
               <tr key={getRowKey(row)}>
                 {columns.map((column) => (
-                  <td key={column.name}>
+                  <td key={column.name} className={column.columnClassName || undefined}>
```

Comment:
This behavior is not managed by the dsfr i guess ? 
What prevent you to do that in your own code like `
<div className="myclass">
  <Table .../>
</div>


.my-class {
  td {
    ...
  }
}`

Reply:
Our client, "**Ministère de la transition écologique et de la cohésion des territoires**" ask us to change **td** background color depending on cell value.
For example, my request is to set green background color to the td if the state value is "VALIDEE"

Reply:
I wait your invitation to the dsfr slack

Reply:
Can you give me your mail ? (no idea how to share that, so lets do it there and then remove it for privacy reason ?)

Reply:
cyril.billon@keyconsulting.fr